server/peers.py

```diff
@@ -77,8 +77,8 @@ def connection_made(self, transport):
         proto_ver = (version.PROTOCOL_MIN, version.PROTOCOL_MAX)
         self.send_request(self.on_version, 'server.version',
                           [version.VERSION, proto_ver])
-        self.send_request(self.on_peers_subscribe, 'server.peers.subscribe')
         self.send_request(self.on_features, 'server.features')
+        self.send_request(self.on_peers_subscribe, 'server.peers.subscribe')
 
```

Comment:
Understood.  The problem is that there is no guaranteed order of replies (even if that's what it looks like most of the time) and I don't want that to be an implicit assumption for the code to work properly.  The sends are guaranteed to go out in the order written but the remote end is free to reply in any order.
So in my push I did it so that we wait until verification from prior replies and then send the request.